nixos/modules/services/backup/borgbackup.nix

```diff
@@ -112,7 +116,7 @@ let
   mkBorgWrapper = name: cfg: mkWrapperDrv {
     original = "${pkgs.borgbackup}/bin/borg";
     name = "borg-job-${name}";
-    set = { BORG_REPO = cfg.repo; } // (mkPassEnv cfg) // cfg.environment;
+    set = mkBorgEnvironment cfg;
```

Comment:
This doesn't always make sense: What if the user that runs the wrapper doesn't have permission to execute `passCommand`?

Reply:
Same question applies to `BORG_REPO`/`BORG_RSH` - what if the user that runs the wrapper can't access the ssh key?
The wrapper script isn't meant to work when run by any user - that's not its purpose.
Users would have to run it as `config.services.borgbackup.jobs.<name>.user`, which is fine.